Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -17,11 +17,20 @@ public class PlayerManager : MonoBehaviour
 
     public int chips;
 
-    private FPSInputManager fpsInput;
+    public FPSInputManager fpsInput;
 
     [SerializeField]
     private GameObject meshBase;
 
+    [SerializeField]
+    private List<Item> items;
+
+    public void PerformTransaction(Item item, int cost)
+    {
+        items.Add(item);
+        chips -= cost;
```

Comment:
Perhaps we should perform *some* sanity check here, instead of naively subtracting the cost from our chip count?

Reply:
We might need to consult @Stektpotet regarding this, as it's only imported code.

I do however think this code is subject to change in #44 , so I'll await this I think

Reply:
Sounds like we should put this off, yeah.